apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -303,12 +303,13 @@ function useInitNavLinks() {
 
     console.log(`[NAV] listening for deep links, account ${account.name}`);
     deepLinkInitialised = true;
-    getInitialURLOrTag(false).then((url) => {
+    getInitialDeepLink().then((url) => {
       if (url == null) return;
       handleDeepLink(nav, url);
     });
 
-    addEventListener("url", ({ url }) => handleDeepLink(nav, url));
+    const sub = addEventListener("url", ({ url }) => handleDeepLink(nav, url));
+    return () => sub.remove();
```

Comment:
just to double check this doesn’t break deep links if you switch tabs to another one right?

Reply:
just tested--works